lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -279,40 +279,40 @@ module.exports = [
     url: 'http://localhost:10200/online-only.html',
     audits: {
       'is-on-https': {
-        score: true,
+        score: 1,
       },
       'uses-http2': {
-        score: false,
+        score: 0,
       },
       'external-anchors-use-rel-noopener': {
-        score: true,
+        score: 1,
       },
       'appcache-manifest': {
-        score: true,
+        score: 1,
       },
       'geolocation-on-start': {
-        score: true,
+        score: 1,
       },
       'link-blocking-first-paint': {
-        score: 100,
+        score: 1.00,
```

Comment:
mixed `1.00`s and `1`s. I'd personally rather just embrace it and go with all `1` all the time, but should at least be consistent in a file and/or context

Reply:
I personally feel the same about the trailing 0s on e.g. `0.80`, too. Why hold on to the digit if we're not going to be doing 0-100 :)

Reply:
very well! :)
fixed up all these. also standardized on a leading 0, so we don't have a mix of `.25` and `0.25`